pkgs/tools/security/gnupg/22.nix

```diff
@@ -69,6 +69,9 @@ stdenv.mkDerivation rec {
 
     # add gpg2 symlink to make sure git does not break when signing commits
     ln -s $out/bin/gpg $out/bin/gpg2
+
+    # Make libexec tools available in PATH
+    ln -s -t $out/bin $out/libexec/*
```

Comment:
You could separate this change out into it's own PR, to give this a better chance of being merged, since I'm assuming this is what is causing the hundreds of rebuilds

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/100031, thanks.